apps/cnspec/cmd/bundle.go

```diff
@@ -30,6 +30,7 @@ func init() {
 	policyBundlesCmd.AddCommand(policyLintCmd)
 
 	// fmt
+	policyFmtCmd.Flags().StringSlice("sort", []string{}, "sort the selected fields in the bundle.")
```

Comment:
If I read this right, I cannot just sort with `--sort` but need to understand what I want to sort?